tests/integration_tests.rs

```diff
@@ -272,6 +272,19 @@ fn squeeze_limit_line_numbers() {
         .stdout("   1 line 1\n   2 \n   3 \n   4 \n   5 line 5\n   6 \n   7 \n   8 \n   9 \n  10 \n  20 line 20\n  21 line 21\n  22 \n  23 \n  24 line 24\n  25 \n  26 line 26\n  27 \n  28 \n  29 \n  30 line 30\n");
 }
 
+#[test]
+fn list_themes() {
+    bat()
+        .arg("--color=always")
+        .arg("--list-themes")
+        .assert()
+        .success()
+        .stdout(predicate::str::contains("DarkNeon").normalize())
+        .stdout(predicate::str::contains("Monokai Extended").normalize())
+        .stdout(predicate::str::contains("Output the square of a number.").normalize())
+        .stdout(predicate::str::contains(" (default)").normalize());
```

Comment:
I would expect a single `contains` that contains "Monokai Extended (default)".

Right now your test will pass even if the wrong theme is marked as default, right?

Reply:
@Enselic You're right. However, the default theme could be `Monokai Extended Light` instead of `Monokai Extended` for MacOS with preferences setup to [light mode](https://github.com/sharkdp/bat/blob/e8d777b73a270330e240a64635851e2342c56323/src/assets.rs#L103) (according to [macos_dark_mode_active()](https://github.com/sharkdp/bat/blob/e8d777b73a270330e240a64635851e2342c56323/src/assets.rs#L403)).

Several ways to fix it are possible:
1. no change: the check is kept weak.
2. assert "Monokai Extended (default)". The test will break on some configuration. It could pass currently the CI, and, one day, will break because the MacOS image used has changed preferences setup.
3. assert "Monokai Extended (default)" but ignore the test on MacOS with `cfg_attr()`. It's future proof but not tested on MacOS.
4. assert a regexp like "Monokai Extended.* (default)". 
5. assert "Monokai Extended (default)" and mock `macos_dark_mode_active()` to be sure the function returns `true`.
6. other way I didn't think about?

The regexp solution (4) is my preferred one.

Which solution do you want I implement?

Reply:
If the default ends up different on macOS, I think it makes sense to `[cfg(...)]` the expected default based on platform.

Reply:
On MacOS, the default can be `Monokai Extended` or `Monokai Light Extended` (if some preferences are defined).

Do you mean you prefer solution 3?
Or doing 3 and adding another test only for macOS?

Reply:
In CI I'm pretty sure the default on macOS is always the same? If that unexpectedly changes, I think it is good if we are notified in the form of a failing test.

So I don't think you need to ignore the test on macOS, I think it should work to cfg the expected default theme.

Reply:
The macOS output in CI is different from other operating systems so I used `cfg` attribute. Is it good for you or did you expect [`cfg!` macro](https://doc.rust-lang.org/rust-by-example/attribute/cfg.html)?

A bold style is applied to the theme name. The `\x1B[0m` characters are the removal of this bold style.